demisto_sdk/commands/generate_docs/common.py

```diff
@@ -196,9 +196,10 @@ def execute_command(command_example, insecure: bool):
             if entry.contents:
                 content = entry.contents
                 if isinstance(content, STRING_TYPES):
-                    md_example += content
+                    # The replacement of <br> by <br/> is for compatibility with our docs site
+                    md_example += content.replace('<br>', '<br/>')
```

Comment:
dont we also need to replace `<BR>` ?

Reply:
Is there anything like this ?!
will ask Francesco